cmd/subnetcmd/list.go

```diff
@@ -45,7 +46,7 @@ func listSubnets(cmd *cobra.Command, args []string) error {
 	table.SetAutoMergeCells(true)
 	table.SetRowLine(true)
 
-	files, err := os.ReadDir(app.GetBaseDir())
+	subnets, err := os.ReadDir(filepath.Join(app.GetBaseDir(), constants.SubnetDir))
```

Comment:
should we make this an `app` function? `app.GetAllSubnets()`?

Reply:
We actually have `GetSidecarNames` there, which does nearly the same - instead of returning all subdirs of `{cli-root}/subnets`, "just" those which actually have a sidecar inside - which should be the same set really.